Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -3710,12 +3819,17 @@ def fix_array_value(match: Match[str]) -> str:
 
         context.append(alert)
 
+    SEVERITY_STATUSES_REVERSE = {value: key for key, value in SEVERITY_STATUSES.items()}
+    STATUS_PROGRESS_REVERSE = {value: key for key, value in STATUS_PROGRESS.items()}
+
+    ALERT_OR_ISSUE = "Alert" if not is_platform() else "Issue"
     human_readable = [
         {
-            "Alert ID": alert.get("internal_id"),
+            f"{ALERT_OR_ISSUE} ID": alert.get("internal_id"),
             "Detection Timestamp": timestamp_to_datestring(alert.get("source_insert_ts")),
             "Name": alert.get("alert_name"),
-            "Severity": alert.get("severity"),
+            "Severity": SEVERITY_STATUSES_REVERSE.get(alert.get("severity")),
+            "Status": STATUS_PROGRESS_REVERSE.get(alert.get("status.progress")),
```

Comment:
let's do this only if is_platform() to preserve the current behavior in xdr-get-alerts